# For some reason the hardening flags from fedora rpmbuild now need a build
# with -fPIC. Also true for AC_PROG_CXX. Two changes to Makefile.am and
# configure.ac.

diff --git a/Makefile.am b/Makefile.am
index ff13879..1893025 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -8,7 +8,7 @@ AM_CPPFLAGS = -DDEBUG -g -Wall \
             -I$(top_srcdir)/src/mediaserver/cdplugins \
             -DDATADIR=\"${pkgdatadir}\" -DCONFIGDIR=\"${sysconfdir}\"
 
-AM_CXXFLAGS = -std=c++14
+AM_CXXFLAGS = -std=c++14 -fPIC
 
 bin_PROGRAMS = upmpdcli scctl
 
